data/profile.d/fedora-workstation.conf

```diff
@@ -22,6 +22,9 @@ hidden_spokes =
     NetworkSpoke
     PasswordSpoke
     UserSpoke
+hidden_webui_pages =
+    anaconda-screen-accounts
```

Comment:
Do we need the prefix? Will it be a situation when it is not `anaconda-screen` prefix?

If yes, could you please document it somewhere (commit message for example)?

Reply:
This has been discussed more in the web UI PR https://github.com/rhinstaller/anaconda-webui/pull/551#discussion_r1895768401

Reply:
I see, ok.

Reply:
I would still like to see the reasoning as part of the commit if possible.